lib/tasks/openapi_generate.rake

```diff
@@ -0,0 +1,553 @@
+class OpenapiGenerator
+  require 'json'
+
+  PARAMETERS_PATH = "/components/parameters".freeze
+  SCHEMAS_PATH    = "/components/schemas".freeze
+
+  def path_parts(openapi_path)
+    openapi_path.split("/")[1..-1]
+  end
+
+  def openapi_file
+    # TODO(lsmola) how does topo API loads the version?
+    # @openapi_file ||= Pathname.new(__dir__).join("../../public/doc/openapi-3-v#{api_version}.json").to_s
+    @openapi_file ||= Pathname.new(__dir__).join("../../../topological_inventory-ingress_api/public/doc/openapi-3-v0.0.2.json").to_s
```

Comment:
So we had to remove the code to auto-load the api version because it used rails routes, we should add a parameter to this rake task to specify the version or autodetect the latest one, but I'm okay with this for now.